packages/@okta/vuepress-site/docs/guides/custom-widget-gen3/main/index.md

```diff
@@ -77,6 +77,98 @@ See [Customization examples](#customization-examples) for snippets that you can
 
    > **Note:** To discard your changes without publishing them, click **Revert changes** or click the **Code editor** toggle again. Turning off the code editor restores the default JavaScript code.
 
+### Use afterRender with the third generation
+
+The third generation of the Sign-In Widget is built on React.js. It interprets the `afterRender` function in a way that's incompatible with customizations. The Sign-In Widget reverts your customizations to Okta default settings.
+
+Typically, with the second generation of Sign-In Widget, you can use `afterRender` to change label text or input labels on a page. The Sign-In Widget triggers `afterRender` when transitioning to a new page and animations have finished. See [afterRender](https://github.com/okta/okta-signin-widget?tab=readme-ov-file#afterrender).
```

Comment:
```suggestion
With the second generation of Sign-In Widget, you can use `afterRender` to change label text or input labels on a page. The Sign-In Widget triggers `afterRender` when transitioning to a new page and animations have finished. See [afterRender](https://github.com/okta/okta-signin-widget?tab=readme-ov-file#afterrender). This will not work in the third generation of Sign-in Widget.
```